b2b-sample/package.json

```diff
@@ -65,5 +66,9 @@
     "regenerator-runtime": "0.13.7",
     "rsuite": "^5.22.2",
     "tslib": "^2.3.0"
+  },
+  "engines": {
+    "node": ">=16",
+    "pnpm": ">=8"
```

Comment:
```suggestion
    "node": ">=16.16.0",
    "pnpm": ">=8.11.0"
```

Reply:
Addressed with https://github.com/wso2/samples-is/pull/479/commits/c71b6bb1113eee4b954359517f7aa0b9c94e2423